test/Microsoft.AspNet.Mvc.FunctionalTests/TagHelperSampleTest.cs

```diff
@@ -8,6 +8,8 @@
 using System.Threading.Tasks;
 using Microsoft.AspNet.Builder;
 using Microsoft.AspNet.TestHost;
+using Microsoft.Framework.DependencyInjection;
```

Comment:
slightly prefer leaving the tag helper sample alone since testing here only confirms a page or two can be downloaded.  also trying to minimize test scenarios that will only work with an in-memory test service.